process/transaction/shardProcess.go

```diff
@@ -169,35 +169,38 @@ func (txProc *txProcessor) ProcessTransaction(tx *transaction.Transaction) (vmco
 		if errors.Is(err, process.ErrInsufficientFunds) {
 			receiptErr := txProc.executingFailedTransaction(tx, acntSnd, err)
 			if receiptErr != nil {
+				// TODO: check if OK and receiptError is correct
 				return 0, receiptErr
 			}
 		}
 
 		if errors.Is(err, process.ErrUserNameDoesNotMatchInCrossShardTx) {
 			errProcessIfErr := txProc.processIfTxErrorCrossShard(tx, err.Error())
 			if errProcessIfErr != nil {
+				// TODO: check if OK and errProcessIfErr is correct
```

Comment:
it doesn't matter - these are critical errors.

Reply:
ok